fastlane/Fastfile

```diff
@@ -658,8 +664,13 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Downloads translated metadata from GlotPress"
   lane :download_metadata_strings do |options|
-    app = options[:app] || 'wordpress'
-    paths = APP_SPECIFIC_VALUES[app.to_sym]
+    download_wordpress_metadata_strings(options)
+    download_jetpack_metadata_strings(options)
```

Comment:
@oguzkocer While double-checking the scenarios and the potential impact of this PR on it, I just realized I missed a change here to make this lane work more smoothly for you 🤔 

Basically, since the `finalize_release` lane is still in a state where you need to call it with the `app:` parameter (even if that's gonna change in the future), that means that for now this lane still calls `download_metadata_strings(app: app, version: version["name"], build_number: version["code"])` on line `382` of the `Fastfile` from within that `finalize_release` lane. But that refactor I did here means that this `app:` parameter is now unused on `download_metadata_strings`, so regardless of the app you're finalizing the release for, it will update both metadata.

That might not be a big deal (I think at worse it would just do extra empty commits? so 🤷 ), so it should not really be a blocker. But maybe to make things more consistent we could do a quick change of this lane to this before merging, so that if an `app:` parameter is passed it will only do the app you ask for, and if you don't pass any `app:` parameter (which is the future goal once I get to refactor `finalize_release` in a later PR), it will do both.

```suggestion
    download_wordpress_metadata_strings(options) unless options[:app] != 'wordpress'
    download_jetpack_metadata_strings(options) unless options[:app] != 'jetpack'
```

Of course we will still want to clean that up and refactor this even better in a later PR (probably when I'll get to working on `finalize_release` and uniformize/clean everything up), but in the meantime especially for this to be redady for when you'll submit next Friday, that could be an acceptable patch.

Reply:
I've applied your suggested change, thank you 🙇